helixtrajectorycpp/src/include/trajectory/HolonomicTrajectorySample.h

```diff
@@ -31,8 +32,16 @@ class HolonomicTrajectorySample {
 template<>
 struct fmt::formatter<helixtrajectory::HolonomicTrajectorySample> {
     template<typename ParseContext>
-    constexpr auto parse(ParseContext& ctx);
+    constexpr auto parse(ParseContext& ctx) {
```

Comment:
This function can just be `constexpr auto parse(fmt::format_parse_context& ctx)` without the template, as per https://fmt.dev/latest/api.html?highlight=formatter#formatting-user-defined-types.

Reply:
Should be fixed